vopono_core/src/network/wireguard.rs

```diff
@@ -166,14 +184,14 @@ impl Wireguard {
 
         NetworkNamespace::exec(
             &namespace.name,
-            &["ip", "link", "add", &if_name, "type", "wireguard"],
+            &["ip", "link", "add", &if_name, "type", &ip_link_type],
         )?;
 
         NetworkNamespace::exec(
             &namespace.name,
-            &["wg", "setconf", &if_name, "/tmp/vopono_wg.conf"],
+            &[&executable_wg, "setconf", &if_name, "/tmp/vopono_wg.conf"],
         )
-        .context("Failed to run wg setconf - is wireguard-tools installed?")?;
+        .context(format!("Failed to run {executable_wg} setconf - is wireguard-tools installed?"))?;
```

Comment:
In this place `wireguard-tools` is printed. Does it matter? Adding an extra parameter to `run()` just to print the correct package name seems odd.

Reply:
I think it's okay, anyone manually using Amnezia would know anyway.